packages/mask/src/plugins/EVM/contexts/index.tsx

```diff
@@ -0,0 +1,8 @@
+import type { FC } from 'react'
+import { TokenPickerProvider } from './TokenPicker'
+
+export * from './TokenPicker'
+
+export const EvmContextProvider: FC = ({ children }) => {
+    return <TokenPickerProvider>{children}</TokenPickerProvider>
+}
```

Comment:
We might create and add `<NFTPickerProvider />` here as well.

Reply:
Do we need to create such a Provider for each chain in the future?

Reply:
Maybe. 🤔